src/lib/config.ts

```diff
@@ -47,3 +47,11 @@ export const siteMode = (({ VITE_SITE_MODE = "local" }) => {
   }
   return VITE_SITE_MODE;
 })(import.meta.env);
+
+
+/**
+ * See isChainInMaintenance()
+ */
+export const chainsUnderMaintenance = {
+    "binance": "Binance"
+};
```

Comment:
(my suggested change) – slight duplication of data to include the human-readable chain name, but saves unnecessary API call and allows us to isolate `checkChainMaintenance` to `[chain]/__layout.svelte`